cli/dcoscli/package/main.py

```diff
@@ -308,8 +313,14 @@ def _install(package_name, options_path, app_id, cli, app, yes):
               "repositories"
         raise DCOSException(msg)
 
-    # TODO(CD): Make package version to install configurable
-    pkg_version = pkg.latest_version()
+    if sw_version:
```

Comment:
This function is already too long.  Can you decompose this into a `_get_package_version()` function?